config/packages/doctrine_migrations.yaml

```diff
@@ -3,10 +3,4 @@ doctrine_migrations:
         # namespace is arbitrary but should be different from App\Migrations
         # as migrations classes should NOT be autoloaded
         'DoctrineMigrations': '%kernel.project_dir%/migrations'
-    enable_profiler: '%kernel.debug%'
-
-    # This is required to ensure BC with previous table name
-    # https://github.com/doctrine/migrations/blob/3.0.x/UPGRADE.md
-    storage:
-        table_storage:
-            table_name: migration_versions
```

Comment:
Won't this cause issues on existing deployments?

Reply:
It will, restored.